llm4pddl/approaches/llm_open_loop_approach.py

```diff
@@ -59,24 +59,34 @@ def solve(self, task: Task) -> Tuple[Optional[Plan], TaskMetrics]:
         return self._llm_responses_to_plan(responses, task)
 
     def train(self, dataset: Dataset) -> None:
-        self._create_prompt_prefix(dataset)
+        self._create_prompt_prefix(dataset, FLAGS.random_object_names)
```

Comment:
minor: let's change `random_object_names` to `use_random_object_names` to make clear that it's a bool and not some collection of random object names